torch/csrc/utils/python_dispatch.cpp

```diff
@@ -105,91 +105,86 @@ struct EnableHermeticPyObject {
   bool old_python_snapshot_;
 };
 
-class PythonKernelHolder : public c10::OperatorKernel {
-  c10::SafePyObject func_;
-  c10::DispatchKey dispatch_key_;
+PythonKernelHolder::PythonKernelHolder(
```

Comment:
We move the `PythonKernelHolder` declaration from the source file to the header file. W/ this change, we have to refine the `PythonKernelHolder` a little bit. From the implementation perspective, we do NOT change any implementation logic but just the code style and the declaration and definition per class member function.